common-modules/src/main/java/com/softserve/if072/common/model/User.java

```diff
@@ -11,15 +15,23 @@
 
     private int id;
     private String name;
+
+    @NotEmpty(message = "{error.email.notnull}")
+    @Size(min = 5, max = 64, message = "{error.email.size}")
     private String email;
+
+    @NotEmpty(message = "{error.password.notnull}")
```

Comment:
If you are going to use Hibernate validator, better to use @NotBlank, it consider "   " as empty.